src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -362,21 +367,18 @@ + (JavaComponentAccessibility *) createWithParent:(JavaComponentAccessibility *)
 
     // otherwise, create a new instance
     JavaComponentAccessibility *newChild = nil;
-    newChild = [CommonComponentAccessibility getComponentAccessibility:javaRole];
-    if (newChild == nil) {
-        if ([javaRole isEqualToString:@"pagetablist"]) {
-            newChild = [TabGroupAccessibility alloc];
-        } else if ([javaRole isEqualToString:@"table"]) {
-            newChild = [TableAccessibility alloc];
+    if ([javaRole isEqualToString:@"pagetablist"]) {
+        newChild = [TabGroupLegacyAccessibility alloc];
+    } else if ([javaRole isEqualToString:@"table"]) {
```

Comment:
Do we even need legacy peer implementation in the new classes?

Reply:
But when are these Legacy classes used? The "pagetablist" and "table" roles are being handled in commonComponentAccessibility and new implementation classes will be used there. Why can't we just remove these legacy classes and any references to them, instead of renaming them

Reply:
By agreement, it was decided that we do not remove the legacy code yet until at least one release has passed to make sure there are no regressions.
The legacy code can then be removed with a separate commit without any difficulty.